README.md

```diff
@@ -343,24 +352,3 @@ Now of course you could implement this all in a client side library, but for the
 ## The Conclusion
 
 This has been a fantastic little experiment to have worked on. Even better it's been successful. I can claim that because it's used in at least one production system. Taking some excellent tools (ZMQ mostly) and building a new tool to help others build yet more tools is a very rewarding experience. If you think you may be interested in building a project/service/tool using this work, let us know! If you find something wrong submit an issue or better yet pull request a fix!
-
```

Comment:
Looks like my editor cleaned up some extra whitespace. Let me know if this an issue and I can revert those changes.